soap/soap.d.ts

```diff
@@ -8,13 +8,19 @@
 declare module 'soap' {
     import * as events from 'events';
 
-    class WSSecurity {
+    class Security {
```

Comment:
**Secuity** is base class, for  this interface _setSecurity(s: Security): void;_ ,it can support more security. For example: WsSecurity, SSLSecurity. url:https://github.com/vpulim/node-soap#security